umap/settings/__init__.py

```diff
@@ -43,3 +43,6 @@
                     globals()["STATICFILES_DIRS"].insert(0, value)
                 else:
                     globals()[key] = value
+
+# Expose these settings for consumption by e.g. django.settings.configure.
+settings_as_dict = {k: v for k, v in globals().items() if k.isupper()}
```

Comment:
I believe it could be done on a separate pull request, would that work for you?

Reply:
OK, up to you. I think it's just a matter of copy pasting the ws server code in a file at the right place and using the Command class, and this may make your life easier to run the websocket server during the tests (but may also not :p ).

Reply:
I've updated the pull request to run the WebSocket server via a django management command (it was easier than expected !).

See commit [`38d19db` (#1754)](https://github.com/umap-project/umap/pull/1754/commits/38d19dbf2755e8d34160d149add6e331897d6e9a) for more info.